docs/reference/experiment-config.txt

```diff
@@ -1223,6 +1223,27 @@ for more details on how to set environment variables in containers.
    The file system path to use as the mount point in the trial runner
    container for storing the local cache.
 
+***********
```

Comment:
This is WIP, I won't land without cleaning it up/clarifying based on what we actually land in the other PRs.